src/components/TemporaryDrawer/index.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { withStyles } from '@material-ui/core/styles';
+import Drawer from '@material-ui/core/Drawer';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import MailIcon from '@material-ui/icons/Mail';
+import AnnouncementIcon from '@material-ui/icons/Announcement';
+import TrendingUpIcon from '@material-ui/icons/TrendingUp';
+import EventsIcon from '@material-ui/icons/Event';
+
+import { toggleDrawer } from '../../services/ui/actions';
+
+const styles = {
+  list: {
+    width: 250,
+  },
+};
+
+class TemporaryDrawer extends React.Component {
+
+  // Switch statement used in render to assign icons
+  renderSwitch(text) {
+    switch(text) {
+      case 'Stats':
+        return <TrendingUpIcon />;
+      case 'Send email':
+        return <MailIcon />;
+      case 'Events':
+        return <EventsIcon />;
+      case 'Announcements':
+        return <AnnouncementIcon />;
+      default:
+        return null;
+    }
+  }
+
+  render() {
+    const { classes } = this.props;
+
+    return (
+      <Drawer open={this.props.isOpen} onClose={() => this.props.toggle()} anchor="left">
+        <div className={classes.list}>
+          <List>
+            {['Stats', 'Send email', 'Events', 'Announcements'].map((text) => (
```

Comment:
I feel like generating these from a list isn't the best approach since you need that switch statement to determine which icon to render. One approach is to write out everything by hand. Another one might be to create another component for creating these items, passing in the text as a prop, and passing the Icon through `props.children`